guides/_subsections/add-a-trusted-certificate.md

```diff
@@ -0,0 +1,40 @@
+## [Add a Trusted Certificate](#-add-a-trusted-certificate)
+If you have Java Management Extensions (JMX) over SSL enabled on your containerized JVMs, you must configure Cryostat to trust the SSL certificate presented by the containerized JVM when Cryostat attempts to open a JMX connection. If you do not complete this configuration, Cryostat cannot open a JMX connection for the purposes of performing JFR management tasks.
+
+Here's how to add a trusted SSL certificate with the Cryostat web UI.
+
+<ol>
+    <li>
+        {% include howto_step.html
+          summary="Navigate to the Security tab"
+          image-name="store-jmx-credentials-1.png"
+          text="
+            <ol>
+                <li>
+                    Click the <i>Security</i> tab.
+                </li>
+                <li>
+                    Click the <i>Upload</i> button on the <i>Import SSL Certificates</i> card. This action opens a file-upload dialog, where you can choose the certificate that you want to upload to Cryostat. You can repeat this process multiple times to add multiple trusted certificates.
```

Comment:
Do you think we should include a screenshot for the dialog or would it be a too extra?

Reply:
Sure, added.